### PR TITLE
Cash Advance changes updated with limitations

### PR DESCRIPTION
**Pull request by GaganSinghArora**

### CODE REVIEW

**Comment by prdarm** [comment]
bank20210217195249346

**Review by prdarm — CHANGES_REQUESTED** [review]
https://mobidrive.com/sharelink/6Cvfj9qiMKk4K5jOZkIcPt5wih0KXsTBvanivCxrHugl

File: src/api-reference/cash-advance/v4.cash-advance.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ layout: reference
 ---
 {% include prerelease.html %}
 
-The Cash Advance API can be used to create, view, and issue cash advance.
+The Cash Advance API can be used to create, view, and issue a cash advance.
```
Comment (prdarm) [inline src/api-reference/cash-advance/v4.cash-advance.md:7]: https://mobidrive.com/sharelink/6Cvfj9qiMKk4K5jOZkIcPt5wih0KXsTBvanivCxrHugl

**Review by prdarm — APPROVED** [review]

File: src/api-reference/cash-advance/v4.cash-advance.md (hunk 1 above)
Comment (prdarm) [reply src/api-reference/cash-advance/v4.cash-advance.md:7]: src/api-reference/cash-advance/v4.cash-advance.md

**Review by prdarm — APPROVED** [review]
src/api-reference/cash-advance/v4.cash-advance.md

**Review by prdarm — APPROVED** [review]

**Review by prdarm — APPROVED** [review]
src/api-reference/cash-advance/v4.cash-advance.md

**Review by prdarm — APPROVED** [review]

**Comment by jonikamelcher** [comment]
:shipit:

**Pull request merged**